README.md

```diff
@@ -1,72 +1,41 @@
 # Microsoft Azure Active Directory Authentication Library (ADAL) for Python
+=====================================
```

Comment:
The markdown format here isn't exactly right.

A generic tips: you can visualize the markdown locally with some plugins in your browser. Or if you already commit and push, you can [visualize it right from github](https://github.com/AzureAD/azure-activedirectory-library-for-python/blob/nc-readme-updates/README.md).

Reply:
Thanks for catching that.